scenarios/command/README.md

```diff
@@ -201,3 +201,14 @@ To run the dotnet sample execute each line below in a different shell/terminal.
 ```bash
  dotnet/command_consumer/bin/Debug/net7.0/command_consumer --envFile=mobile-app.env
 ```
+
+### python
```

Comment:
I think this is required as well to be able to run this:
https://github.com/Azure-Samples/MqttApplicationSamples/blob/main/scenarios/telemetry/README.md?plain=1#L260-L267